website/docs/ReactNavigation.md

```diff
@@ -198,6 +202,155 @@ describe('Testing react navigation', () => {
 });
 ```
 
+## Drawer Navigator
+
+Testing the Drawer Navigation requires an additional setup step for mocking the Reanimated library.
+
+### Setting up
+
+Install the packages required for React Navigation. For this example, we will use a [drawer navigator](https://reactnavigation.org/docs/drawer-navigator/) to transition between a home screen and an additional screen.
+
+```
+$ yarn add @react-native-community/masked-view @react-navigation/native @react-navigation/drawer react-native-gesture-handler react-native-reanimated react-native-safe-area-context react-native-screens
+```
+
+Create a [`./DrawerAppNavigator.js`](https://github.com/callstack/react-native-testing-library/blob/master/examples/reactnavigation/src/DrawerAppNavigator.js) component which will list the navigation stack:
+
+```jsx
+import 'react-native-gesture-handler';
+import React from 'react';
+import {createDrawerNavigator} from '@react-navigation/drawer';
+
+const { Screen, Navigator } = createDrawerNavigator();
```

Comment:
Better to use the style followed in official docs without destructring

Reply:
@satya164 I copied the way the existing example is built: https://github.com/callstack/react-native-testing-library/blob/master/website/docs/ReactNavigation.md